application/F3DStarter.cxx

```diff
@@ -992,6 +996,31 @@ int F3DStarter::Start(int argc, char** argv)
       }
     }
 
+    // Process Command Script file
+    const std::string& commandScriptFile = 
+      this->Internals->AppOptions.CommandScriptFile;
+    if (!commandScriptFile.empty())
+    {
+      std::ifstream scriptFile(commandScriptFile);
+      if (scriptFile.is_open())
+      {
+        std::string command;
+        while (std::getline(scriptFile, command))
+        {
+          if (!command.empty() && command[0] != '#')
```

Comment:
comment support should be implemented in `utils::tokenize`. Feel free to not do it and plan it for later, but we should not support comment here.